addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/RFXComBindingConstants.java

```diff
@@ -64,6 +64,8 @@
             BRIDGE_RFXTRX315, BRIDGE_RFXREC443);
 
     // List of all Channel ids
+    public final static String CHANNEL_RAW_MESSAGE = "rawMessage";
+    public final static String CHANNEL_RAW_PAYLOAD = "rawPayload";
```

Comment:
I think that gets mapped in undecoded.xml:
		<channels>
			<channel id="rawMessage" typeId="rawmessage" />
			<channel id="rawPayload" typeId="rawpayload" />
		</channels>